index.html

```diff
@@ -91,6 +92,7 @@ <h2>Release Calendar</h2>
 				</td>
 				<td>
 					<ul>
+						<li><a href="/releases/kovan/1.9.3/index.html">Release 1.9.3</a> (Wednesday, 2021-04-08)</li>
```

Comment:
Date seems wrong...maybe it should be `2021-08-04` instead?

Reply:
Fixed.